src/redux/book/book.js

```diff
@@ -1,51 +1,70 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-
+const DISPLAY_BOOKS = 'bookstore/books/DISPLAY_BOOKS';
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/sF3v4K9XvF0U28c60dQV/books';
 // actions
-export const addBook = (book) => ({
-  type: ADD_BOOK,
-  book,
-});
-export const removeBook = (id) => ({
-  type: REMOVE_BOOK,
-  id,
-});
-
-const initialBookState = [
-  {
-    id: 1,
-    genre: 'Novel',
-    title: 'Anna Karenina',
-    author: 'Lev Tolstoy',
-  },
-  {
-    id: 2,
-    genre: 'Novel',
-    title: 'One Hundred Years of Solitude',
-    author: 'Gabriel García Márquez',
-  },
-  {
-    id: 3,
-    genre: 'Dystopian fiction',
-    title: 'Nineteen Eighty-Four',
-    author: 'George Orwell',
+
+// export const displayBooks = createAsyncThunk(
+//   DISPLAY_BOOKS,
+//   async () => axios.get(URL).then((response) => {
+//     const objData = response.data;
+//     const books = Object.keys(objData).map((key) => ({
+//       id: key, ...objData[key][0],
+//     }));
+//     return books;
+//   }),
+// );
```

Comment:
Please kindly remove unused code for it makes your code base look untidy. Microverse follows a clean code policy so it would be nice if you removed the commented code. Thank you 😄